dev/index.html

```diff
@@ -7,7 +7,10 @@
     <div class="container" />
 
     <script>
-      TeamContributionCalendar.default(".container", ["tccemptyuser"])
+      TeamContributionCalendar.default(".container", 
+      [["gomorizsolt", ['2019-07-01','2019-08-04']], "tccemptyuser1"], 
```

Comment:
I want to avoid `Array.isArray` etc calls. I'd propose an object format instead:

```suggestion
      [{ name: "gomorizsolt", from: '2019-07-01', to: '2019-08-04' }, "tccemptyuser1"], 
```

This way we could still support the old string input format as `object['name'] || object`.

Let's mention the new format in the readme and the old one as deprecated.

Reply:
> the old one as deprecated.

I don't get this. What's the purpose of making it deprecated? Why not just supporting both formats? Because of the additional logic, i.e. this way we need to differentiate the two cases?

Reply:
I mean it's much more convenient in the sense that there's no need to define an object when you'd like to use the package in its simplest form(i.e. aggregate the calendars without specifying a certain timeframe).

Reply:
> Why'd one support multiple formats when the object representation is clear enough?

So we don't introduce a breaking change. We can support a deprecated format and remove it eventually.

Reply:
> So we don't introduce a breaking change. We can support a deprecated format and remove it eventually.

Yea, agreed. I was trying to figure out why we wouldn't like to keep both formats(i.e. without making one deprecated and removing it later).

Reply:
It would be of course simple to have a single format in the future.

Reply:
Gotcha. I think we can resolve this thread then(the format has already been modified to your proposal except there's no option to define an end date because of https://github.com/c-hive/team-contribution-calendar/pull/64#discussion_r399778117).